dhash/module.py

```diff
@@ -383,16 +400,16 @@ async def _report_duplicate(
         original: The original attachment.
         distance: Hamming distance between the original and repost.
         """
-        tx = TranslationContext(message.guild.id, message.author.id)
+        (tc,) = TranslationContext(message.guild.id, message.author.id)
```

Comment:
I don't understand, why `(tc,)` instead of `tc`? `TranslationContext` does not return a tuple.